pkg/agent/flannel/setup.go

```diff
@@ -133,7 +133,7 @@ func createFlannelConf(nodeConfig *config.Node) error {
 			return err
 		}
 	case config.FlannelBackendWireguard:
-		backendConf = wireguardBackend
+		backendConf = strings.Replace(wireguardBackend, "%flannelConfDir%", filepath.Dir(nodeConfig.FlannelConf), -1)
```

Comment:
```suggestion
		backendConf = strings.ReplaceAll(wireguardBackend, "%flannelConfDir%", filepath.Dir(nodeConfig.FlannelConf))
```

Reply:
Nice, thanks for the suggestion. I'll change the config.FlannelBackendIPSEC case in another PR to use ReplaceAll too